WRO-697: Use babel-preset-enact's babel config (#38)
enactjs/storybook-utils: CHANGELOG.md

## Unreleased

> * Updated `babel.js` to use `babel-preset-enact` that supporting own enact app config.

Reviewer: ```suggestion
* Updated `babel.js` to use `babel-preset-enact` module that holds the babel config for Enact.
```
